pkg/reconciler/kubernetes/tektonhub/transformer.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2021 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tektonhub
+
+import (
+	"strings"
+
+	networkingv1 "k8s.io/api/networking/v1"
+
+	mf "github.com/manifestival/manifestival"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func injectOwner(owner []v1.OwnerReference) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		kind := u.GetKind()
+		if kind == "CustomResourceDefinition" {
+			return nil
+		}
+		u.SetOwnerReferences(owner)
+		return nil
+	}
+}
+
+func changeNamespace(targetNamespace string) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		if u.GetNamespace() != targetNamespace {
+			u.SetNamespace(targetNamespace)
+			return nil
+		}
+		return nil
+	}
```

Comment:
do we really need to redefine these 2 transformers as they already exist in manifestival and is used everywhere else.

- https://github.com/tektoncd/operator/blob/e37aa26ed3db9b58dd3f1777ff8ae775cfe604d8/pkg/reconciler/common/transformers.go#L100

- https://github.com/tektoncd/operator/blob/e37aa26ed3db9b58dd3f1777ff8ae775cfe604d8/pkg/reconciler/common/transformers.go#L65